tests/framework/clients/helm/helm.go

```diff
@@ -0,0 +1,150 @@
+package helm
+
+import (
+	"context"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var helmCmd = "helm_v3"
+
+func InstallRancher(restConfig *rest.Config) error {
```

Comment:
Can we get comments on all the export function? For our godocs

Reply:
Done.